src/mixins/embed.js

```diff
@@ -80,35 +82,51 @@ export default {
     const { id } = params;
     let { type = '' } = query;
     const { referrer = '' } = query;
+    const { iscn_id: iscnId } = query;
```

Comment:
How about checking `params.id` with ISCN ID regex, so no need to get from the query string?

Reply:
Discussed with Wei & William, I prefer to use the previous URL format, in ISCN version `params.id` 'iscn' will be paired with query sicn_id.